pkgs/applications/editors/quartus-prime/default.nix

```
@@ -110,10 +138,41 @@ in buildFHSUserEnv {
     xorg.libXrender
   ];
 
-  extraInstallCommands = ''
+  passthru = {
+    inherit quartus;
```

Usually this is what's done:

```suggestion
    unwrapped = quartus;
```

See Neovim's derivation for reference.